apps/judicial-system/backend/src/app/modules/case/case.controller.ts

```diff
@@ -548,6 +548,7 @@ export class CaseController {
   @RolesRules(
     prosecutorRule,
     prosecutorRepresentativeRule,
+    publicProsecutorStaffRule,
```

Comment:
Shouldn't the public prosecutors staff have access to other generated files for indictments?

Reply:
Yes they should, good catch thank you! Will fix